core/src/main/java/apoc/nodes/Nodes.java

```diff
@@ -189,6 +189,7 @@ public boolean connected(@Name("start") Node start, @Name("start") Node end, @Na
         if (nodes == null || nodes.isEmpty()) return Stream.empty();
         if (nodes.size() == 1) return Stream.of(new VirtualPathResult(nodes.get(0), null, null));
         Set<Node> nodeSet = new LinkedHashSet<>(nodes);
+        config.putIfAbsent("selfRel", false);
```

Comment:
could you elaborate why we need a different default here (compared to the global one in `RefactorConfig`)?